.env.example

```diff
@@ -2,11 +2,11 @@ NEXTAUTH_SECRET=dAbxJF2DRzqwGYn+BWKdj8o9ieMri4FWsmIRn77r2F8=
 REACT_APP_API_URL=http://localhost:3333/api
 NEXTAUTH_URL=http://localhost:3000
 NEXT_AUTH_API_URL=http://backend:3333/api
-KEYCLOAK_URL=<KEYCLOAK_URL>
-KEYCLOAK_CLIENT_ID=<client_id>
-KEYCLOAK_CLIENT_UUID=<uuid>
-KEYCLOAK_REALM=<realm>
-KEYCLOAK_CLIENT_SECRET=<client_secret>
-KEYCLOAK_ADMIN_CLIENT_ID=ctims-admin # a client with sevice account enabled
-KEYCLOAK_ADMIN_CLIENT_SECRET=<secret>
+KEYCLOAK_URL=<http://localhost:8181>
+KEYCLOAK_CLIENT_ID=ctims
+KEYCLOAK_CLIENT_UUID=15d6ff8e-a439-45c9-8644-615d6f414309
+KEYCLOAK_REALM=ctims
+KEYCLOAK_CLIENT_SECRET=GD0UkeywCNobyDdqmwOiPrHlkMB1uLzO
+KEYCLOAK_ADMIN_CLIENT_ID=ctims
+KEYCLOAK_ADMIN_CLIENT_SECRET=GD0UkeywCNobyDdqmwOiPrHlkMB1uLzO
```

Comment:
Like John mentioned, these should not be commited.

Reply:
I thought I did this but may not have committed this version. Ill remove again